packages/web/pages/docs/rest-api.mdx

```diff
@@ -28,3 +34,10 @@ With the REST API, you can use mailing for email templating even if most of your
 | POST   | /api/lists/[listId]/members            | add a member to a list        |
 | GET    | /api/lists/[listId]/members/[memberId] | get a list member's status    |
 | PATCH  | /api/lists/[listId]/members/[memberId] | update a list member's status |
+
+## API keys
+
+For endpoints that require an API key, the following methods are supported:
+
+- Include the API key in the request headers: `X-API-KEY=...`
+- Include the API key as a query string parameter: `?apiKey=...`
```

Comment:
```suggestion
## API keys

Mailing uses API keys to protect endpoints that consume limited resources (e.g. sending email with your transport).

To get an API key, either [set up a database](https://www.mailing.run/docs/deploy#adding-a-database) and log in (recommended) or set the `MAILING_API_KEY` environment variable in your deployment environment.

You then pass your API key in each request with one of these methods:

- Include the API key in the request headers: `X-API-KEY=...`
- Include the API key as a query string parameter: `?apiKey=...`


```

Reply:
Is it clear enough that you need to make up your own API key if you set MAILING_API_KEY?  We might get questions about where to get a key from

Reply:
👍  edited again for clarity. feel free to edit

Reply:
Nice 👍